linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -135,6 +109,49 @@ private void run(String groupId, String artifactId)
     }
   }
 
+  private static ImmutableList<String> coordinatesList(List<Artifact> artifacts) {
+    return artifacts.stream().map(Artifacts::toCoordinates).collect(toImmutableList());
+  }
+
+  /**
+   * Returns an error message if {@code snapshotSymbolProblems} contains new problems compared with
```

Comment:
This is an odd pattern. Perhaps we should split the detection of the error from the formatting of the message. Or perhaps the comment isn' correct? I don't see this method returning null.

Reply:
The javadoc was outdated. Initially I was thinking to mix error detection and formatting message but it's already separated.